### PR TITLE
feat(extension): connect layout

### PR DESCRIPTION
**Pull request by vladdezhniuk**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/ConnectDapp/Connected.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  animation: ${showingFrames} 2s ease-in-out;
+  color: ${(props) => props.theme.white};
+`
+export const Connected: React.FC<{
+  setConnectStep: React.Dispatch<React.SetStateAction<ConnectStep>>
+}> = ({ setConnectStep }) => {
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setConnectStep('InitialScreen')
+      window.location.replace('/#/plugin/coinslist')
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/ConnectDapp/Connected.tsx:22]: use router

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/ConnectDapp/Confirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+}> = ({ setConnectStep }) => {
+  const MOCK_SITE_URL = 'app.uniswap.org'
+
+  const connect = () => {
+    setConnectStep('Connectiing')
+  }
+
+  const deny = () => {
+    setConnectStep('InitialScreen')
+    window.location.replace('/#/plugin/coinview')
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/ConnectDapp/Confirmation.tsx:121]: use router

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/ConnectDapp/Connecting.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  animation: ${siteIconFrames} 2s ease-in both;
+`
+const ConnectingWrapper = styled(Flex)`
+  width: 100%;
+  justify-content: space-between;
+`
+export const Connecting: React.FC<{
+  setConnectStep: React.Dispatch<React.SetStateAction<ConnectStep>>
+}> = ({ setConnectStep }) => {
+  useEffect(() => {
```
Comment (NikolaiYurchenko) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/ConnectDapp/Connecting.tsx:37]: please add empty lines before return

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  children,
+  onPrefixClick,
+  onSuffixClick,
+  prefix,
+  suffix
+}: {
+  children?: JSX.Element
+  onPrefixClick?: () => void
+  onSuffixClick?: () => void
+  prefix?: JSX.Element
```
Comment (NikolaiYurchenko) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx:30]: can them really be undefined?

Comment (NikolaiYurchenko) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx:30]: and maybe it is a good idea to make Ptops type

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/CoinsList/CoinsListHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+          <Text size='12px'>ETH</Text>
+          <span />
+        </WalletSymbol>
+      </WalletAddress>
+    </DefaultHeader>
+  )
+}
+
+const mapStateToProps = (state) => ({
+  walletAddress: selectors.core.kvStore.eth.getDefaultAddress(state).getOrElse('')
```
Comment (NikolaiYurchenko) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/CoinsList/CoinsListHeader.tsx:65]: where do we use this selector in CoinsListHeader?

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  suffix?: JSX.Element
+}) => {
+  return (
+    <Wrapper justifyContent='space-between'>
+      <PrefixSuffix onClick={onPrefixClick}>{prefix}</PrefixSuffix>
+      {children}
```
Comment (NikolaiYurchenko) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx:36]: add tab

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx (hunk 4 above)
Comment (vladdezhniuk) [reply packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx:30]: Thats not my component) merge mistake

**Pull request closed without merging**